Libraries/src/Amazon.Lambda.Logging.AspNetCore/LambdaILogger.cs

```diff
@@ -29,53 +29,74 @@ public bool IsEnabled(LogLevel logLevel)
 				_options.Filter(_categoryName, logLevel));
 		}
 
-		public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
-		{
-			if (formatter == null)
-			{
-				throw new ArgumentNullException(nameof(formatter));
-			}
-
-			if (!IsEnabled(logLevel))
-			{
-				return;
-			}
-
-			// Format of the logged text, optional components are in {}
-			//  {[LogLevel] }{ => Scopes : }{Category: }{EventId: }MessageText {Exception}{\n}
-
-			var components = new List<string>(4);
-			if (_options.IncludeLogLevel)
-			{
-				components.Add($"[{logLevel}]");
-			}
-
-			GetScopeInformation(components);
-
-			if (_options.IncludeCategory)
-			{
-				components.Add($"{_categoryName}:");
-			}
-			if (_options.IncludeEventId)
-			{
-				components.Add($"[{eventId}]:");
-			}
-
-			var text = formatter.Invoke(state, exception);
-			components.Add(text);
-
-			if (_options.IncludeException)
-			{
-				components.Add($"{exception}");
-			}
-			if (_options.IncludeNewline)
-			{
-				components.Add(Environment.NewLine);
-			}
-
-			var finalText = string.Join(" ", components);
-			Amazon.Lambda.Core.LambdaLogger.Log(finalText);
-		}
+        public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        {
+            if (formatter == null)
+            {
+                throw new ArgumentNullException(nameof(formatter));
+            }
+
+            if (!IsEnabled(logLevel))
+            {
+                return;
+            }
+
+            var lambdaLogLevel = ConvertLogLevel(logLevel);

```

Comment:
nit: move this closer to where it is being used

Reply:
Done